src/Symfony/Component/Notifier/Bridge/Smsbox/composer.json

```diff
@@ -25,7 +25,9 @@
     ],
     "require": {
         "php": ">=8.2",
+        "symfony/clock": "^6.4|^7.0",
         "symfony/http-client": "^6.4|^7.0",
+        "symfony/intl": "^6.4|^7.0",
```

Comment:
why do we need this one?

Reply:
To use the `Countries` class in SmsboxOptions, this ease the check of the country code validity

Reply:
let's make it optional and skip the validation is the class isn't found?